lnbits/app.py

```diff
@@ -262,7 +262,7 @@ async def check_installed_extension_files(ext: InstallableExtension) -> bool:
 
     zip_files = glob.glob(os.path.join(settings.lnbits_data_folder, "zips", "*.zip"))
 
-    if f"./{str(ext.zip_path)}" not in zip_files:
+    if f"./{ext.zip_path!s}" not in zip_files:
```

Comment:
was ist das?

Reply:
!s 	Interpolates the string representation from the .__str__() method
:D